src/components/pages/error/ErrorPage.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Link } from "react-router-dom";
+
+export default function ErrorPage() {
+  return (
+    <div id="error-page">
```

Comment:
```suggestion
    <div>
```

ici, pas besoin de rajouter d'id, surtout s'il n'apporte rien en plus. Cela vient "polluer" ta lisibilité de la partie affichage du composant